stdpopsim/catalog/ApiMel/species.py

```diff
@@ -0,0 +1,117 @@
+import stdpopsim
+
+from . import genome_data
+
+# [The following are notes for implementers and should be deleted
+#  once the recombination rates have been inserted]
+# This is the per-chromosome recombination rate, typically the mean
+# rate along the chromosome.
+# Values in this dictionary are set to -1 by default, so you have
+# to update each one. These should be derived from the most reliable
+# data and how they were obtained should be documented here.
+# The appropriate citation must be added to the list of
+# recombination_rate_citations in the Genome object.
+
+_BeyeEtAl = stdpopsim.Citation(
+    doi="https://dx.doi.org/10.1101%2Fgr.5680406",
+    year=2006,
+    author="Beye et al.",
+    reasons=stdpopsim.CiteReason.REC_RATE,
```

Comment:
Hi @janaobsteter. The reason the tests are failing is because the "reasons" fields for the citations needs to be a `set` of CiteReasons. So above, this needs to be `reasons={stdpopsim.CiteReason.REC_RATE}`, with curly braces. Similarly for the other `reasons=` lines.

Reply:
Cool, thanks! Will correct this!